python/ray/data/_internal/stats.py

```diff
@@ -1,4 +1,5 @@
 import collections
+from dataclasses import dataclass
 import time
 from contextlib import contextmanager
 from typing import Dict, List, Optional, Set, Tuple, Union
```

Comment:
For the below `DatasetStatsSummary.extra_metrics` type annotation nit.
```suggestion
from typing import Dict, List, Optional, Set, Tuple, Union, Any
```